Sources/Widgets/Widgets3D/PolyLineWidget/state.js

```diff
@@ -8,7 +8,7 @@ export default function generateState() {
       mixins: ['origin', 'color', 'scale1', 'visible'],
       name: 'moveHandle',
       initialValues: {
-        scale1: 0.1,
+        scale1: 0.05,
```

Comment:
it would be good to add some explanation about this magic number

Reply:
When interpreted as a display scale for SphereHandleRepresentation, it's a fraction of the vertical height of the viewport. I'll add a comment about it.